src/buildCommon.js

```diff
@@ -116,13 +139,19 @@ var mathit = function(value, mode, options, classes) {
     if (/[0-9]/.test(value.charAt(0)) ||
             // glyphs for \imath and \jmath do not exist in Math-Italic so we
             // need to use Main-Italic instead
-            utils.contains(mainitLetters, value) ||
-            utils.contains(greekCapitals, value)) {
-        return makeSymbol(
-            value, "Main-Italic", mode, options, classes.concat(["mainit"]));
+            utils.contains(mainitLetters, value)) {
+            // For greek capitals, we have a choice, but Math-Italic
+            // seems closer to LaTeX.
+            //|| utils.contains(greekCapitals, value)) {
```

Comment:
I meant to ask you about this.  Yes, it seems so:

Greek capital letters are in both fonts, `Main-Italic` and `Math-Italic`.  So which should we use? The [master code](https://github.com/Khan/KaTeX/blob/master/src/buildCommon.js#L120) claims to use `Main-Italic` for Greek capital letters.  But if you try the [demo](https://khan.github.io/KaTeX/) with `\mathit{\Omega\imath}`, and inspect, you'll see that the `\Omega` gets class `mathit` while `\imath` gets `mainit`:

```html
<span class="mord mathit" style="margin-right: 0.05017em;">Ω</span>
<span class="mord mainit">ı</span>
```

In fact, the master code ends up calling `mathit()` with a single-unicode-character string (`Ω`), while `greekCapitals` is defined in terms of the macro name (`"\Omega"`).  When my code fixed this, I ended up changing Greek capital letters to use `Main-Italic` font, which caused some tests to fail.  I confirmed with `texcmp` that `Math-Italic` seems slightly closer, so kept it there.

Let me know if you'd like this code changed at all...